cloud-docs/run-environment: Don't overload the term "run" (#638)
hashicorp/terraform-docs-common: website/docs/cloud-docs/run/run-environment.mdx

Terraform Enterprise instances must have network connectivity to any connected VCS providers or managed infrastructure providers.

> <a name="concurrency-and-run-queuing"></a>

Reviewer: ```suggestion
<a name="concurrency-and-run-queuing"></a>
```
We can do this in another PR, but instead of adding this anchor, we should search for the links to this heading update them.

## Concurrency and Job Queuing

> HCP Terraform uses multiple concurrent worker VMs, which take jobs from a global queue of operations that are ready to execute. (This includes both creating new plans and then applying those plans once they've been approved.)

Reviewer: ```suggestion
HCP Terraform uses multiple concurrent worker VMs, which take jobs from a global queue of operations that are ready to execute. This includes both creating new plans and then applying those plans once they've been approved.
```
Per the style guide, avoid using parentheses in prose.

If the global queue has more jobs than the workers can handle at once, some of them must wait until a worker becomes available. When the queue is backed up, HCP Terraform gives different priorities to different kinds of jobs:

- Applying approved plans has the highest priority.
- Creation of normal plans has the next-highest priority.
- Creation of speculative plans has the lowest priority.

> HCP Terraform can also delay some jobs to make performance more consistent across organizations. If an organization requests a large number of jobs at once, HCP Terraform queues some of them immediately, and delays the rest until some of the initial batch have finished; this allows every organization to continue executing jobs even during periods of especially heavy load.

Reviewer: ```suggestion
HCP Terraform can also delay some jobs to make performance more consistent across organizations. If an organization requests a large number of jobs at once, HCP Terraform queues some of them immediately and delays the rest until some of the initial batch have finished. Doing so allows every organization to continue executing jobs even during periods of especially heavy load.
```
I don't think the semicolon is wrong here, but I'd rather use two sentences to keep it simple.
